src/trade/trade-chart/TradeViewChart.js

```diff
@@ -60,15 +54,27 @@ export default class TradeViewChart extends PureComponent {
       $(this.barspinner).hide();
     });
   }
+
   destroyChart() {
     this.chart && this.chart.actions.destroy();
     this.chart = null;
   }
 
   componentDidMount() {
-    const trade = this.props.tradeForChart.toJS();
-    this.initChart(trade);
+    (async () => {
```

Comment:
same here

Reply:
huh. Great suggestion @aminroosta . Definitely simplifies the code a bit.